src/clj/puppetlabs/services/ca/certificate_authority_disabled_service.clj

```diff
@@ -21,6 +21,6 @@
     [this localcacrl]
     (log/info "CA disabled; ignoring retrieval of CA CRL"))
 
-  (get-oid-mappings
+  (get-auth-handler
     [this]
-    {}))
+    (fn [request] (wrap-with-authorization-check request {:oid-map {}}))))
```

Comment:
I think the issue with that is that the master, legacy routes, and admin api services all use the authorization middleware regardless of whether or not the disabled CA service is used.